burn-tch/src/ops/int_tensor.rs

```diff
@@ -190,17 +190,30 @@ impl<E: TchElement> IntTensorOps<Self> for LibTorch<E> {
         lhs: TchTensor<i64, D>,
         rhs: TchTensor<i64, D>,
     ) -> TchTensor<i64, D> {
-        TchOps::div(lhs, rhs)
+        let copy = false;
```

Comment:
What does this actually do? seems like you're just forcing the float div to be used?
I took out all the changes in this file, and the failing tests on my repo and here still pass

Reply:
Yeah somehow I had to update this for the CI, I cast the i64 tensor to a f64 tensor without copying the data since it seems there are problems with integer division with tch.